.github/workflows/identity_pull_request.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Build base branch
         run: ./gradlew :identity-example:assembleRelease && mv identity-example/build/outputs/apk/theme1/release/identity-example-theme1-release.apk identity-example/build/outputs/apk/theme1/release/identity-example-release-base.apk
       - name: Upload APK
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
There were some behavior changes between 3 and 4 that break what we're doing here. I think this will need to remain on 3. Or we will have to rewrite how this works.